jar-infer/jar-infer-lib/src/main/java/com/uber/nullaway/jarinfer/BytecodeAnnotator.java

```diff
@@ -260,6 +260,7 @@ public static void annotateBytecodeInAar(
               inputJarEntry, jarIS, jarOS, nonnullParams, nullableReturns, stripJarSignatures);
           inputJarEntry = jarIS.getNextJarEntry();
         }
+        jarOS.close();
```

Comment:
@navahgar Didn't want to make a full commit for just this one, but this was causing `classes.jar` files to fail on `unzip` (strangely `jar` itself can extract them, which is why our tests couldn't catch it). Not sure how to test for it, so I am just sneaking the fix into this PR 🤷‍♂ 

See. https://coderanch.com/t/590754/java/Editing-jar-archive-Java-application

Reply:
Oh, thats interesting. Thanks for fixing that.